### PR TITLE
User avatar

### PR DESCRIPTION
**Pull request by bwencke**

### CODE REVIEW

**Comment by bwencke** [comment]
Probably want to actually get user data before merging this.

File: imports/ui/pages/users/UserPage.jsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import React, { Component } from 'react'
 
+import UserCard from '/imports/ui/components/users/UserCard'
+
+const test_user = {
+  avatarUrl: 'http://www.thewrap.com/wp-content/uploads/2015/11/Donald-Trump.jpg',
```
Comment (garr741) [inline imports/ui/pages/users/UserPage.jsx]: Uhh no.

**Review by garr741 — CHANGES_REQUESTED** [review]

**Review by garr741 — APPROVED** [review]

File: imports/ui/pages/users/UserPage.jsx [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+  name: "John Smith",
+}
+
 const styles = {
   twoColumnLayout: {
     display: "flex",
```
Comment (ashleyzeg) [inline imports/ui/pages/users/UserPage.jsx]: single quotes?

**Review by ashleyzeg — APPROVED** [review]

**Comment by bwencke** [comment]
@garr741 @ashleyzeg @mhoc I changed this PR a lot since you guys approved it, so if you want to look at it again, that would be awesome. Also, "UserCard" will be moved once Ashley finished the about me and skills components.

**Review by ashleyzeg — APPROVED** [review]

**Pull request merged**